### DISCUSSION TITLE
Does the persistBy option do anything at all right now? It doesn't look like it.

### DISCUSSION THREAD

**Discussion by vesper8** [CodeDredd/pinia-orm#1912] [General]
@CodeDredd Unless I'm missing something, according to this search for the keyword: https://github.com/search?q=repo%3ACodeDredd%2Fpinia-orm+persistBy&type=code

It doesn't look like persistBy does anything at all? A remnant from vuex-orm, now a dead body? Or am I missing something?

**Comment by CodeDredd** [comment]
You are right. It does nothing. I forgot to remove it or let's say add it back later. Because i dont't have an update method. There is a PR for the update method but reserved it for v2.
